dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/relationship/RelationshipRequestParams.java

```diff
@@ -93,4 +93,7 @@ public class RelationshipRequestParams implements PageRequestParams {
 
   @OpenApi.Property(value = String[].class)
   private List<FieldPath> fields = FieldFilterParser.parse(DEFAULT_FIELDS_PARAM);
+
+  @OpenApi.Property(defaultValue = "false")
```

Comment:
default values are now automatically picked up. I think you don't even need the annotation at all here.